spec/end-to-end-tests/tests/create.js

```diff
@@ -182,19 +182,6 @@ var test_create_add_file_twice = function (browser) {
 module.exports = {
   'Create a dataset, happy path': test_create_happy_path,
 //  'Create a dataset, invalid title': test_create_invalid_title,
```

Comment:
Yes, I'll remove it. I'd left it there to remind me about discussing what's a valid dataset name, but I'll record that as an issue instead.